tests/integration/istioio/security/mtls_migration_test.go

```diff
@@ -0,0 +1,176 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package security
+
+import (
+	"testing"
+
+	"istio.io/istio/pkg/test/env"
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/istioio"
+)
+
+//https://istio.io/docs/tasks/security/mtls-migration/
+//https://github.com/istio/istio.io/blob/release-1.2/content/docs/tasks/security/mtls-migration/index.md
+func TestMutualTLSMigration(t *testing.T) {
+	t.Skip("https://github.com/istio/istio/issues/17909")
+
+	framework.
+		NewTest(t).
+		Run(istioio.NewBuilder().
+			Add(istioio.Command{
+				Input: istioio.Inline{
+					FileName: "create_ns_foo_bar_legacy.sh",
+					Value: `
+$ kubectl create ns foo
```

Comment:
Can we make these values string literals and reference them? Would make it easier to read.

Reply:
I'm not sure that would help with readability ... we'd effectively be moving them to the top of the file somewhere, so you'd have to go back and forth between the definition and the call site. If we were going to do that we might as well just use input files.  Just my thoughts.

@geeknoid WDYT?

Reply:
Input files work here as well. My thought is more that as a reader/writer, most of the content follows the same indentation/styles except for these which reset the indentation. Was just trying to figure out if there's a way we could keep the same style. Using inlining you also lose editor assistance such as auto-completion/syntax highlighting/etc. Just a couple of nits.

Reply:
Understood. And as discussed offline, it's entirely up to the author how they want to manage this content. The `Input` supports files as well as inline.